libfwupdplugin/fu-device.c

```diff
@@ -2200,6 +2242,8 @@ fu_device_add_string (FuDevice *self, guint idt, GString *str)
 		fu_common_string_append_kv (str, idt + 1, "LogicalId", priv->logical_id);
 	if (priv->proxy != NULL)
 		fu_common_string_append_kv (str, idt + 1, "Proxy", fu_device_get_id (priv->proxy));
```

Comment:
I think that you should change this to `ProxyID` or `ProxyDeviceID` to disambiguate now.

Reply:
Done.